roles/sfs/templates/ats-shared-fs.sh.j2

```diff
@@ -8,4 +8,7 @@ fi
 {% for key, value in sfs_environment.items() %}
 export {{ key }}="{{ value | join(' ') }}"
 {% endfor %}
-${JAVA_HOME}/bin/java ${JAVA_OPTS} -jar ${ATS_HOME}/alfresco-shared-file-store-*.jar > {{ logs_folder }}/ats-shared-fs.log
+${JAVA_HOME}/bin/java ${JAVA_OPTS} \
+	-jar ${ATS_HOME}/alfresco-shared-file-store-*.jar \
+	--spring.config.additional-location=optional:sfs-mtls.properties \
+	> {{ logs_folder }}/ats-shared-fs.log
```

Comment:
I think there is some systemd feature to redirect logs to a specific file, so we can also add an exec prefix to the command and handle termination better, right now is probably getting killed abruptly after a stop timeout

Reply:
created OPSEXP-2068